OptimizelySDK/Config/DatafileProjectConfig.cs

```diff
@@ -135,6 +135,20 @@ public enum OPTLYSDKVersion
             = new Dictionary<string, Dictionary<string, Variation>>();
         public Dictionary<string, Dictionary<string, Variation>> VariationKeyMap { get { return _VariationKeyMap; } }
 
+        /// <summary>
+        /// Associative array of experiment ID to associative array of variation key to variations
+        /// </summary>
+        private Dictionary<string, Dictionary<string, Variation>> _VariationKeyMapByExperimentId
+            = new Dictionary<string, Dictionary<string, Variation>>();
+        public Dictionary<string, Dictionary<string, Variation>> VariationKeyMapByExperimentId { get { return _VariationKeyMap; } }
```

Comment:
Shouldn't it return _VariationKeyMapByExperimentId, instead of _VariationKeyMap?